Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -103,10 +102,32 @@ public extension CBPurchase {
     }
 
     //Buy the product
+    @available(swift, deprecated: 4.0, message: "This will be removed in upcoming versions, Please use this API func purchaseProduct(product: CBProduct, customer : CBCustomer? = nil, completion) .")
```

Comment:
Please set the appropriate version here.

Reply:
But what version we should keep  when we don't have particular version that it will get removed.